### DISCUSSION TITLE
Packages/com.unity.recorder/manual/RecordingPlayMode

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#568] [Notes]
# Packages/com.unity.recorder/manual/RecordingPlayMode

https://docs.unity3d.com/Packages/com.unity.recorder@latest/index.html?subfolder=%2Fmanual%2FRecordingPlayMode.html

**Comment by TristanM42** [comment]
warning, not capping fps in recorder gives an extremely slow playback (recorded with approximately 130 fps)
